README.md

```diff
@@ -603,15 +613,75 @@ output file can be specified by using the `--out` flag.
-to point to these databases.
-
-When provisioning a sandbox from the GitHub pull request, you can override the
-default by including `(ephemeral databases)` or `(persistent databases)` on the
-same line as the sandbox domain in the pull request description. For example:
+By default, instances will use local, ephemeral databases that are destroyed
+when app servers belonging to an instance are terminated. If you want to use
+external databases that can be used by any app server belonging to an instance,
+follow these steps:
+
+1. Change the `INSTANCE_EPHEMERAL_DATABASES` setting to False. Note that this is
+   only necessary if you want instances to use persistent databases by default.
+   If you only want a specific instance to use persistent databases, simply set
+   the value of the `use_ephemeral_databases` field to `True` and save the instance
+   (cf. below).
+
+2. Set up external mysql and mongo databases, making a note of hostname
+   and authentication information (username, password) for each one of them.
+
+3. In your `.env` file, set `DEFAULT_INSTANCE_MYSQL_URL` and `DEFAULT_INSTANCE_MONGO_URL`
+   to URLs that point to the MySQL and MongoDB servers created in the previous step:
+
+   ```
+   DEFAULT_INSTANCE_MYSQL_URL='mysql://<user>:<password>@<hostname>'
+   DEFAULT_INSTANCE_MONGO_URL='mongodb://<user>:<password>@<hostname>'
+   ```
+
+   Note that `<user>` must have necessary permissions to create databases and users,
+   and to grant privileges on the MySQL/MongoDB server. `<hostname>` can be an IP address.
+
+   The next time you create an instance, the instance manager will automatically
+   create a `MySQLServer` and a `MongoDBServer` using the values of the
+   `DEFAULT_INSTANCE_MYSQL_URL` and `DEFAULT_INSTANCE_MONGO_URL` settings
+   and assign it to the instance.
+
+   **Alternatively**, you can skip setting `DEFAULT_INSTANCE_MYSQL_URL` and `DEFAULT_INSTANCE_MONGO_URL`
+   and create `MySQLServer` and `MongoDBServer` objects yourself via the shell or via the Django admin.
+
+   ```python
+   from instance.models.database_server import MySQLServer, MongoDBServer
+
+   MySQLServer.objects.create(
+       hostname='<hostname>',
+       user='<username>',
+       password='<password>',
+       port=<port>,
+   )
+
+   MongoDBServer.objects.create(
+       hostname='<hostname>',
+       user='<username>',
+       password='<password>',
+       port=<port>,
+   )
+   ```
+
+   You can create as many `MySQLServer` and `MongoDBServer` objects as you like.
+   If there are multiple servers of a given type to choose from, the instance manager
+   will randomly select and assign one of them when you create a new instance.
+
+When an app server for an instance that is configured to use persistent databases is provisioned,
+the instance manager will automatically add necessary settings to enable the app server to read
+and store application data on the external database servers that the instance is associated with.
```

Comment:
This sentence is a bit convoluted..  How about this instead?

When the instance manager provisions an app server for an instance that uses persistent databases, it will automatically add the necessary settings on the associated external database server to enable the app server to read and store application data.